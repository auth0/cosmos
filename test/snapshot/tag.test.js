import React from 'react'
import renderer from 'react-test-renderer'
import { Tag } from '@auth0/cosmos'
import 'jest-styled-components'

describe('Tag', () => {
  describe('renders with automation attribute', () => {
    it('single Tag', () => {
      const tree = renderer.create(<Tag />).toJSON()
      expect(tree).toMatchSnapshot()
    })

    it('group of Tags', () => {
      const tree = renderer.create(<Tag.Group />).toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
