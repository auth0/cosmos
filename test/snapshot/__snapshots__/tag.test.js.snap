// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag renders with automation attribute group of Tags 1`] = `
.c0 .c1 {
  margin-right: 8px;
}

.c0 .c1:last-child {
  margin-right: 0;
}

<div
  className="c0"
  data-cosmos-key="tag.group"
/>
`;

exports[`Tag renders with automation attribute single Tag 1`] = `
.c0 {
  display: inline-block;
  font-size: 13px;
  color: #3BC0F2;
  line-height: 1;
  padding: 4px 8px;
  background: #DBF4FD;
  border: 1px solid #3BC0F2;
  border-radius: 3px;
  cursor: inherit;
}

.c0 .sc-bwzfXH {
  cursor: pointer;
  stroke: #3BC0F2;
  margin-left: 8px;
}

<span
  className="c0"
  data-cosmos-key="tag"
/>
`;
